react/features/base/participants/components/ParticipantView.native.js

```diff
@@ -205,7 +212,7 @@ class ParticipantView extends Component<Props> {
         // black screen.
         const waitForVideoStarted = false;
         const renderVideo
-            = !this.props._audioOnly
+            = !this.props._audioOnly && !hideVideo
```

Comment:
I'd put that condition first, also can you destructure _audioOnly, since you are there?